chainerx_cc/chainerx/context.cc

```diff
@@ -1,6 +1,9 @@
 #include "chainerx/context.h"
 
-#ifndef _WIN32
+#ifdef _WIN32
+#define RTLD_LOCAL 0
+#define RTLD_NOW 0x00002
```

Comment:
Could you write a comment above these two lines why they're required (and that they're expected to be temporary)?

Reply:
Rather than defining them here, I think it's better to enclose their usage as an implementation detail of platforms.
In other words, how about removing `flags` argument from `chainerx::DlOpen` and `chainerx::platform::DlOpen`?